### DISCUSSION TITLE
Tuya IoT Platform Service Expiration Notice

### DISCUSSION THREAD

**Discussion by lord-amenegg** [jasonacox/tinytuya#286] [Q&A]
Dear Tinytuya team
I have received the email with the above title yesterday.
As I am not really using their cloud services, can I safely ignore it?

Many thanks
Alex

**Comment by jasonacox** [comment]
Hi @lord-amenegg ,

The test account for the Tuya IoT Platform is confusing.  It's not clear what will happen.

However, it won't impact your local access as long as you already have the local-keys, but it does mean you may not be able to get new local-keys without signing up for another test account.  

To help us determine what Tuya is doing, after your service expires, can you attempt to run wizard to see if it still works?

Also, when you signed up, did you select "Skip this step" on this screen or did you perhaps select "Individual" or "Organization" membership?  I'm trying to confirm if "Skip this step" makes any difference.

![image](https://user-images.githubusercontent.com/836718/220519394-e61abcb4-e486-4248-8f67-71497f507aac.png)

**Comment by lord-amenegg** [comment]
Hi there.
I have tried it today and it seems to continue working.
Not 100% sure if when I did the sign in I skipped the step, but I think I did.
Thanks

**Reply by jasonacox** [reply]
That's good news. Hopefully it will continue to work.

**Reply by Siltm** [reply]
I received the same email and this week when I want to add new devices I can not get the local key, the devices that already existed before are still working without problem, but I can no longer get data from the new devices or control them.

**Reply by lord-amenegg** [reply]
That will explain why i can no longer connect to some of my lights
